main/src/addins/MacPlatform/MainToolbar/StatusBar.cs

```diff
@@ -202,6 +202,7 @@ public StatusBar ()
 					return;
 
 				ReconstructString ();
+				if(Layer.Sublayers != null)
```

Comment:
change looks correct, but formatting is weird no?

At least I think you should indent the *foreach*